internal/tools/pod_exec_cmd.go

```diff
@@ -0,0 +1,159 @@
+package tools
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/strowk/mcp-k8s-go/internal/k8s"
+	"github.com/strowk/mcp-k8s-go/internal/utils"
+
+	"github.com/strowk/foxy-contexts/pkg/fxctx"
+	"github.com/strowk/foxy-contexts/pkg/mcp"
+	"github.com/strowk/foxy-contexts/pkg/toolinput"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+func NewPodExecCommandTool(pool k8s.ClientPool) fxctx.Tool {
+	k8sNamespace := "namespace"
+	k8sPodName := "podName"
```

Comment:
```suggestion
	k8sPodName := "pod"
```

Reply:
I think the name should be same as for pod in other tools (f.e in `pod_logs.go`). I prefer shorter names and descriptions if things are clear enough, since everything that server returns back to client in regards to tools (names, input schema, descriptions) would take from context window, so the longer that stuff is - the less useful input tokens user have then left in their conversation